pipenv/cli.py

```diff
@@ -984,38 +984,26 @@ def sigwinch_passthrough(sig, data):
 ))
 @click.argument('command')
 @click.argument('args', nargs=-1)
-@click.option('--no-interactive', is_flag=True, default=False, help="Run the command in non-interactive mode.")
 @click.option('--three/--two', is_flag=True, default=None, help="Use Python 3/2 when creating virtualenv.")
 @click.option('--python', default=False, nargs=1, help="Specify which version of Python virtualenv should use.")
-def run(command, args, no_interactive=False, three=None, python=False):
+def run(command, args, three=None, python=False):
     # Ensure that virtualenv is available.
     ensure_project(three=three, python=python, validate=False)
 
-    # Automatically enable --no-interactive, when applicable.
-    if not sys.stdout.isatty():
-        no_interactive = True
+    command_path = which(command)
 
-    # Spawn the new process, and interact with it.
-    try:
-        c = pexpect.spawn(which(command), list(args))
-    except pexpect.exceptions.ExceptionPexpect:
-        click.echo(crayons.red('The command ({0}) was not found within the virtualenv!'.format(which(command))))
+    if not os.path.exists(command_path):
+        click.echo(crayons.red('The command ({0}) was not found within the virtualenv!'.format(command_path)))
         sys.exit(1)
 
-    # Windows!
```

Comment:
This is a minor nit, but any reason to move this comment inside the conditional? This project tends to have comments precede the code they're describing.